apps/home/templates/apps_home/blocks/video_block.html

```diff
@@ -0,0 +1,76 @@
+{% load static i18n wagtailimages_tags wagtailembeds_tags block_tags %}
+
+{% get_identifier value as identifier %}
+
+<section class="modul-video ds-block--base">
+    <h2 class="title">{{ value.title }}</h2>
+    <div class="video-container">
+        <div class="block-video__container">
+            <div class="accept_video block-video__video" id="accept-video{{ identifier }}">
+                {% image value.preview_image fill-980x550 as image %}
+                <img
+                  class="preview_image"
+                  src="{{ image.url }}"
+                  alt="{{ image.alt }}"
+                  loading="lazy" />
+                <p class="overlay_text">
+                  {% if 'youtube' in value.video.url %}
+                    {% trans 'By viewing the video, you agree that your data will be transmitted to <strong>YouTube</strong>. For more information see' %}
+                    <a target="_blank" href="https://www.google.de/intl/de/policies/privacy/">{% trans 'Data protection' %}</a>.
+                  {% elif 'vimeo' in value.video.url %}
+                    {% trans 'By viewing the video, you agree that your data will be transmitted to <strong>Vimeo</strong>.For more information see' %}
+                    <a target="_blank" href="https://vimeo.com/privacy">{% trans 'Data protection' %}</a>.
+                  {% else %}
+                    {% trans 'DSGVO compliant videos are not supported by your browser.' %}
```

Comment:
it's probably more accurate to say 'This video cannot be viewed in a dsgvo compliant way, please click the video link if you wish to view? and add a link?

Reply:
swill check wording with Max after their standup

Reply:
it's an edge case as the embed does not allow non youtube/vimeo to be loaded but yes that is better, but was thinking a link would still be good, but will check with max, also I guess the formfield should have helptext too thinking about it